sdk/cli/bin/arv

```diff
@@ -150,14 +160,7 @@ end
 
 def edit_and_commit_object initial_obj, tmp_stem, global_opts, &block
 
-  content = case global_opts[:format]
-            when 'json'
-              Oj.dump(initial_obj, :indent => 1)
-            when 'yaml'
-              initial_obj.to_yaml
-            else
-              abort "Unrecognized format #{global_opts[:format]}"
-            end
+  content = get_obj_content initial_obj, global_opts
```

Comment:
:+1: moving this out to a method, but now that it's further away from its counterpart in "Load the new object" below, perhaps it's a good time to add an "else abort" in the "Load" section too, so if the format lists go out of sync we throw a nice loud exception right away instead of having a mysterious nil floating about.